client/my-sites/upgrades/domain-management/dns/txt-record.jsx

```diff
@@ -55,7 +55,7 @@ const TxtRecord = React.createClass( {
 						name="data"
 						onChange={ this.props.onChange }
 						value={ this.props.fieldValues.data }
-						placeholder={ this.translate( 'e.g. a=b; verification-key=something', { context: 'TXT DNS Record', textOnly: true } ) } />
+						placeholder={ this.translate( 'e.g. %(example)s', { args: { example: 'a=b; verification-key=something' } } ) } />
```

Comment:
I kinda agree with both of you here - this is a technical field and should not have to be translated, but let's change it to something less arbitrary (and looking like normal English) and use, for example, the value that's needed to [configure SPF for Google Apps](https://support.google.com/a/answer/178723?hl=en): `v=spf1 include:_spf.google.com ~all`. Or `google-site-verification=rXOxyZounnZasA8Z7oaD3c14JdjS9aKSWvsR1EbUSIQ` - but that seems too Google-centric (at least for SPF, the beginning part is the same for other hosts).